sage.py

```diff
@@ -30,1228 +30,17 @@
-    "MALWARE-CNC Flame malware connection - /view.php" : MicroAttackStage.COMMAND_AND_CONTROL,
-    "OS-LINUX Linux Kernel keyring object exploit download attempt" : MicroAttackStage.PRIV_ESC,
-    "OS-LINUX Linux kernel ARM put_user write outside process address space privilege escalation attempt" : MicroAttackStage.PRIV_ESC,
-    "OS-LINUX Linux kernel madvise race condition attempt" : MicroAttackStage.PRIV_ESC,
-    "FILE-PDF PDF with click-to-launch executable" : MicroAttackStage.DATA_DELIVERY,
-    "FILE-PDF Adobe Acrobat Reader PDF font processing memory corruption attempt" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "OS-OTHER Bash environment variable injection attempt" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "OS-OTHER Intel x86 side-channel analysis information leak attempt" : MicroAttackStage.DATA_EXFILTRATION,
-    "OS-OTHER Mac OS X setuid privilege esclatation exploit attempt" : MicroAttackStage.PRIV_ESC,
-    "FILE-EXECUTABLE Microsoft Windows Authenticode signature verification bypass attempt" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "FILE-EXECUTABLE Microsoft CLFS.sys information leak attempt" : MicroAttackStage.DATA_EXFILTRATION,
-    "FILE-EXECUTABLE Kaspersky Internet Security kl1.sys out of bounds read attempt" : MicroAttackStage.END_POINT_DOS,
-    "FILE-EXECUTABLE XOR 0xfe encrypted portable executable file download attempt" : MicroAttackStage.DATA_DELIVERY,
-    "FILE-EXECUTABLE Microsoft Windows NTFS privilege escalation attempt" : MicroAttackStage.PRIV_ESC,
-    "INDICATOR-SHELLCODE ssh CRC32 overflow /bin/sh" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "INDICATOR-SHELLCODE possible /bin/sh shellcode transfer attempt" : MicroAttackStage.DATA_DELIVERY,
-    "FILE-FLASH Adobe Flash Player memory corruption attempt" : MicroAttackStage.PRIV_ESC,
-    "FILE-FLASH Adobe Flash Player use after free attempt" : MicroAttackStage.PRIV_ESC,
-    "FILE-IMAGE Oracle Java Virtual Machine malformed GIF buffer overflow attempt" : MicroAttackStage.PRIV_ESC,
-    "FILE-IMAGE Adobe Acrobat TIFF Software tag heap buffer overflow attempt": MicroAttackStage.ARBITRARY_CODE_EXE,
-    "FILE-JAVA Oracle Java privileged protection domain exploitation attempt" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "FILE-JAVA Oracle Java sun.awt.image.ImagingLib.lookupByteBI memory corruption attempt" : MicroAttackStage.SERVICE_SPECIFIC,
-    "FILE-JAVA Oracle Java RangeStatisticImpl sandbox breach attempt" : MicroAttackStage.SERVICE_SPECIFIC,
-    "FILE-JAVA Oracle Java System.arraycopy race condition attempt" : MicroAttackStage.SERVICE_SPECIFIC,
-    "BROWSER-CHROME Apple Safari/Google Chrome Webkit memory corruption attempt" : MicroAttackStage.ARBITRARY_CODE_EXE,
-    "BROWSER-CHROME Google Chrome FileReader use after free attempt" : MicroAttackStage.PRIV_ESC,
-    "BROWSER-CHROME V8 JavaScript engine Out-of-Memory denial of service attempt" : MicroAttackStage.END_POINT_DOS,
-}
-
-
-recent_suricata_alerts = {
```

Comment:
This one is also not used anywhere else in the code

Reply:
Better to keep it regardless.